airflow/plugins/operators/scrape_state_geoportal.py

```diff
@@ -0,0 +1,206 @@
+import gzip
+import logging
+import os
+from typing import ClassVar, List
+
+import pandas as pd  # type: ignore
+import pendulum
+import requests
+from calitp_data_infra.storage import PartitionedGCSArtifact, get_fs  # type: ignore
+from pydantic import HttpUrl, parse_obj_as
+
+from airflow.models import BaseOperator  # type: ignore
+
+API_BUCKET = os.environ["CALITP_BUCKET__STATE_GEOPORTAL_DATA_PRODUCTS"]
+
+
+class StateGeoportalAPIExtract(PartitionedGCSArtifact):
+    bucket: ClassVar[str]
+    execution_ts: pendulum.DateTime = pendulum.now()
+    dt: pendulum.Date = execution_ts.date()
+    partition_names: ClassVar[List[str]] = ["dt", "execution_ts"]
+    root_url: str
+    service: str
+    product: str
+    where: str
+    outFields: str
+    f: str
+    resultRecordCount: int
+
+    @property
+    def table(self) -> str:
+        return self.product
+
+    @property
+    def filename(self) -> str:
+        return self.table
+
+    class Config:
+        arbitrary_types_allowed = True
+
+    def fetch_from_state_geoportal(self):
+        """ """
+
+        logging.info(f"Downloading state geoportal data for {self.product}.")
+
+        try:
+            # Set up the parameters for the request
+            url = self.root_url + self.service
+            validated_url = parse_obj_as(HttpUrl, url)
+
+            params = {
+                "where": self.where,  # You can change this to filter data
+                "outFields": self.outFields,  # Specify the fields to return
+                "f": self.f,  # Format of the response
+                "resultRecordCount": self.resultRecordCount,  # Maximum number of rows per request
```

Comment:
**suggestion:** Using `'geojson'` for `f`. Also removing documentation from here in favor of documenting the options above.
```suggestion
                "where": self.where,
                "outFields": self.outFields,
                "f": "geojson",
                "resultRecordCount": self.resultRecordCount,
```